__tests__/app.test.js

```diff
@@ -7,6 +7,24 @@ const seed = require("./../db/seeds/seed")
 beforeEach(() => seed({ topicData, userData, commentData, articleData }))
 afterAll(() => db.end())
 
+//api
+describe("GET /api", () => {
+    test("status: 200, responds with JSON object containing all available endpoints in the API", () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        })
+        test("status: 404, responds with error for invalid file path", () => {
```

Comment:
this 404 test should be seperate from all other blocks as a spelling mistake is a mistake regardless of the endpoint so isn't specific to any one